build.gradle

```diff
@@ -0,0 +1,65 @@
+apply plugin: 'java'
+apply plugin: 'application'
+apply plugin: 'idea'
+apply from: 'gradle/quality.gradle'
+
+sourceCompatibility = '1.7'
+targetCompatibility = '1.7'
+
+[compileJava, compileTestJava]*.options*.encoding = 'UTF-8'
+
+mainClassName = "org.apache.mesos.hdfs.Main"
+
+group       = "org.apache.mesos"
+version     = "0.0.2-SNAPSHOT"
+
+ext {
+    mesosVer = "0.21.1"
+    slf4jVersion = "1.7.10"
+    logback_classicVersion = "1.1.2"
+    hadoopVersion = "2.4.0"
```

Comment:
We were using 2.5 to be consistent with Myriad. I'd prefer to be consistent with Myriad